lib/cocoapods/installer/target_installer/pod_target_installer.rb

```diff
@@ -98,6 +98,7 @@ def add_files_to_build_phases
           resource_refs = file_accessor.resources.flatten.map do |res|
             project.reference_for_path(res)
           end
+          resource_refs.compact!
```

Comment:
Some resource files in the glob no longer have direct corresponding file references. This includes things like files within an `xcdatamodeld` directory or `xcassets` directory.